build/ci/set-up-ops-manager.sh

```diff
@@ -75,7 +75,7 @@ echo "create project"
 AGENT_API_KEY=$(./bin/mongocli iam projects create myProj -o="go-template={{.AgentAPIKey}}")
 MCLI_PROJECT_ID=$(./bin/mongocli iam project list -o="go-template={{ (index .Results 0).ID }}")
 
-cat <<EOF > "${XDG_CONFIG_HOME}/mongocli.toml"
+cat <<EOF > "${XDG_CONFIG_HOME}/config.toml"
```

Comment:
[q] what about the folder?

Reply:
It looks like when `XDG_CONFIG_HOME` is set for atlascli [we don't try to add the folder to the path ](https://github.com/mongodb/mongocli/blob/f319c546a8f66a1f6d2970da6e6127d758e180f0/internal/config/profile.go#L661). So I assumed that for mcli would be the same. Is that something we want to do for both?

Reply:
oh I think this uncovers a bug on the atlas implementation when `XDG_CONFIG_HOME` is set the config path for both atlas and now mongocli should be `XDG_CONFIG_HOME/{mongocli|atlascli}/config.toml`